person.rb

```diff
@@ -11,6 +11,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
```

Comment:
- [ ] I will encourage you to add rentals for the purpose of setter and setter by using the attr_accessor